docs/03-tutorials/00-application-connectivity/ac-04-register-secured-api.md

```diff
@@ -118,6 +118,44 @@ To create such a Secret, run this command:
 ```bash
 kubectl create secret generic {SECRET_NAME} --from-literal clientId={CLIENT_ID} --from-literal clientSecret={CLIENT_SECRET} -n kyma-integration
 ```
+## Register an OAuth-secured API with token obtained from client certificate-secured endpoint
+
+This is an example of the **service** object for an API secured with OAuth, where the token is fetched from Mutual-TLS secured endpoint:
+
+```yaml
+  - id: {TARGET_UUID}
+    name: my-oauth-service
+    displayName: "My MTLS OAuth Service"    
+    description: "My service"
+    providerDisplayName: "My organisation"
+    entries:
+    - credentials:
+        secretName: {SECRET_NAME}
+        authenticationUrl: {OAUTH_TOKEN_URL}
+        type: OAuthWithCert
+      targetUrl: {TARGET_API_URL}
+      type: API
+```
+
+This is an example of the Secret containing credentials:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {SECRET_NAME}
+  namespace: kyma-integration
+data:
+  clientId: {BASE64_ENCODED_CLIENT_ID}
+  crt: {BASE64_ENCODED_CERTIFICATE}
+  key: {BASE64_ENCODED_PRIVATE_KEY}
+```
+
+To create such a Secret, run this command:
+
+```bash
+kubectl create secret generic {SECRET_NAME} --from-literal clientId={CLIENT_ID} --from-literal crt={CERTIFICATE} --from-literal key={PRIVATE_KEY} -n kyma-integration
```

Comment:
Again, double-checking to be on the safe side: are there any additional parameters needed because of mTLS?

Reply:
Besides provided parameters you can include use some additional headers and request parameters to obtain a token in both cases OAUTH and mtls OAUTH but they are not documented here and I don't remind myself any example of its usage.

Reply:
As in https://github.com/kyma-project/kyma/pull/14401#discussion_r892192458